resources/services/elasticache/gen.hcl

```diff
@@ -0,0 +1,124 @@
+service = "aws"
```

Comment:
Could you please add `//check-for-changes` here so we can check this directory for drift going forward? :bow: